apps/apple/FluentUITester/macos/FluentUITester-macOS/ViewController.m

```diff
@@ -0,0 +1,22 @@
+#import "ViewController.h"
+#import "AppDelegate.h"
+
+#import <React/RCTRootView.h>
+
+@implementation ViewController
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  RCTBridge *bridge = [((AppDelegate *)[NSApp delegate])bridge];
+  RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge moduleName:@"FluentUITester" initialProperties:nil];
+
+  NSView *view = [self view];
+
+  [view addSubview:rootView];
+  [rootView setBackgroundColor:[NSColor windowBackgroundColor]];
+  [rootView setFrame:[view bounds]];
+  [rootView setAutoresizingMask:(NSViewMinXMargin | NSViewMinXMargin | NSViewMinYMargin | NSViewMaxYMargin | NSViewWidthSizable | NSViewHeightSizable)];
```

Comment:
This may have fixed the 1px issue, but none of this code should be necessary. It is perfectly legal for a client of react-native to use an RCTRootView in autolayout and seemingly it works for iOS

Reply:
Not sure I'll be able to solve this issue in this code review, but we could fix up the autogenerated template code at the react-native-macos level in a separate CR, and then propogate the change to here?